Makefile

```diff
@@ -30,7 +31,7 @@ install: $(TARGET) $(DOCDIR)/$(MAN1)
 uninstall:
 	$(RM) $(DESTDIR)$(BINDIR)/$(TARGET)
 	$(RM) $(DESTDIR)$(MANDIR)/man1/$(MAN1)
-	$(RM) -r $(DESTDIR)$(EXAMPLEDIR)
+	$(RM) -r $(SHAREDIR)
```

Comment:
you could "update" your PR, by updating your branch:
 - push a new patch
 - amend/rebase your patchs and do a push with `--force` option